sparktk-core/src/main/scala/org/trustedanalytics/sparktk/saveload/SaveLoad.scala

```diff
@@ -56,21 +58,24 @@ object SaveLoad {
    * @param zipFile the MAR file to be stored
    * @return full path to the location of the MAR file
    */
-  def saveMar(storagePath: String, zipFile: File): String = {
-    if (storagePath.startsWith("hdfs")) {
+  def saveMar(sc: SparkContext, storagePath: String, zipFile: File): String = {
+
+    val protocol = ScoringModelUtils.getProtocol(storagePath)
+
+    if ("file".equalsIgnoreCase(protocol)) {
+      print("Local")
```

Comment:
improve print message.

Does this really mean local?  What about s3? --does it fall under local or hdsf?   Could provide more help in making the right set of options, besides local and hdfs?

Also, this reveals to me that the protocol/path work you've added above is not unique to ScoringModelUtils, but more generally to sparktk saveload.  That logic should move to this more generic location.  What do you think?